pkg/gather/clusterconfig/operators_pods_and_events.go

```diff
@@ -37,6 +38,9 @@ type CompactedEventList struct {
 	Items []CompactedEvent `json:"items"`
 }
 
+// Used to detect the possible stack trace on logs
+var stackTraceRegex = regexp.MustCompile(`((.*)(/{0,9}){0,9}).([0-9]{0,9})`)
```

Comment:
I am not very good in the regex expressions TBH, but this seems too wide. It's matching logs without any stacktrace from what I've seen.

Reply:
You can try to degrade the monitoring operator (as we do in our tests) and run your code against it. You will see that a lot of logs match your expression, but there is no stacktrace present.